source/MRViewer/MRViewportGL.cpp

```diff
@@ -60,21 +60,6 @@ void ViewportGL::init()
 
     if ( !Viewer::constInstance()->isGLInitialized() )
         return;
-
-    // lines 
-    GL_EXEC( glGenVertexArrays( 1, &add_line_vao ) );
-    GL_EXEC( glGenBuffers( 1, &add_line_vbo ) );
-    GL_EXEC( glGenBuffers( 1, &add_line_colors_vbo ) );
-
-    // points 
-    GL_EXEC( glGenVertexArrays( 1, &add_point_vao ) );
-    GL_EXEC( glGenBuffers( 1, &add_point_vbo ) );
-    GL_EXEC( glGenBuffers( 1, &add_point_colors_vbo ) );
-
-    // border 
-    GL_EXEC( glGenVertexArrays( 1, &border_line_vao ) );
-    GL_EXEC( glGenBuffers( 1, &border_line_vbo ) );
-
     inited_ = true;
```

Comment:
this should be used to know if we have gl or not in others functions, so no need it `isGLInitialized` check each time